torch/csrc/jit/codegen/cuda/type_inference.cpp

```diff
@@ -15,10 +15,64 @@ namespace cuda {
 
 namespace {
 
+at::ScalarType toAccumulateType(const TensorTypePtr& op) {
```

Comment:
Any reason that we are not using `at::toAccumulateType` from `aten/src/ATen/AccumulateType.h`?

Reply:
No good reason I guess.